lib/completion-templates.ts

```diff
@@ -47,3 +47,23 @@ _{{app_name}}_yargs_completions()
 compdef _{{app_name}}_yargs_completions {{app_name}}
 ###-end-{{app_name}}-completions-###
 `;
+
+export const completionFishTemplate = `### {{app_name}} completion - begin. generated by omelette.js ###
```

Comment:
Although we may well wish to acknowledge Omelette somewhere, it seems misleading to say "generated by omelette.js" when it was generated by Yargs?